src/validation.h

```diff
@@ -54,14 +55,14 @@ static const bool DEFAULT_WHITELISTRELAY = true;
 /** Default for DEFAULT_WHITELISTFORCERELAY. */
 static const bool DEFAULT_WHITELISTFORCERELAY = true;
 /** Default for -minrelaytxfee, minimum relay fee for transactions */
-static const CAmount DEFAULT_MIN_RELAY_TX_FEE = COIN / 1000;
+static const CAmount DEFAULT_MIN_RELAY_TX_FEE = RECOMMENDED_MIN_TX_FEE / 10;
```

Comment:
Arguably isn't this backwards, as in isn't the design that ` RECOMMENDED_MIN_TX_FEE`  is 10x `DEFAULT_MIN_RELAY_TX_FEE`?

Reply:
Not really. The minimum relay fee should be lower than the recommended fee, so that the recommendation can be lowered.

I formulated this in the fee proposal as:

> Optimal forward flexibility can be assured by always setting minimal relay to be lower than the recommended fee whenever the default fee recommendation changes.

Reply:
Cool, both work